pandas/core/tools/datetimes.py

```diff
@@ -38,7 +39,7 @@ def _guess_datetime_format_for_array(arr, **kwargs):
         return _guess_datetime_format(arr[non_nan_elements[0]], **kwargs)
 
 
-def _maybe_cache(arg, format, cache, tz, convert_listlike):
```

Comment:
The tz here was not used?

Reply:
tz was passed into the `convert_listlike` function further down, but now I am embedding it into `convert_listlike` with `functools.partial` in `to_datetime`